src/payment/strategies/adyenv2/adyenv2-initialize-options.ts

```diff
@@ -30,12 +30,12 @@ export default interface AdyenV2PaymentInitializeOptions {
      * @deprecated
      * Use additionalActionOptions instead as this property will be removed in the future
      */
-    threeDS2Options: AdyenThreeDS2Options;
+    threeDS2Options: ThreeDS2Options;
 
     /**
      * A set of options that are required to initialize additional payment actions.
      */
-    additionalActionOptions: AdyenAdditionalActionOptions;
+    additionalActionOptions: AdditionalActionOptions;
```

Comment:
Can you please keep the `Adyen` prefix? This is because all the interfaces eventually get exported under the same namespace. So it's better to keep the prefix to avoid possible name collision?